functions/src/api/v1/rooms/service/createTwilioClientToken.ts

```diff
@@ -0,0 +1,23 @@
+import { Room } from '../../../../types/Room'
+import { UID } from '../../../../types/uid'
+import * as AccessToken from 'twilio/lib/jwt/AccessToken'
+import { getTwilioEnv } from '../../../../firebase/env'
+
+const VideoGrant = AccessToken.VideoGrant
+export const TTL_ACCESS_TOKEN_PARTICIPANT_SECONDS = 1440 // 24 hours
```

Comment:
The [docs](https://www.twilio.com/docs/video/tutorials/user-identity-access-tokens#time-to-live) suggest:

> Time-To-Live (ttl)
> Access Tokens must be valid while joining a Room and when reconnecting to a Room due to network disruption or handoff. Therefore, we recommend that you set the ttl to the maximum allowed session length, which is currently 14,400 seconds (4 hours).